### ISSUE TITLE
firebase-admin package missing

### ISSUE THREAD

**Issue by andyfangdz** [tehsis/webtaskio-canirequire#38]
`firebase-admin` is very useful when building a serverless app with Firebase and Webtask.

https://www.npmjs.com/package/firebase-admin

**Comment by kaizau** [comment]
Agreed! The existing [firebase packages](https://tehsis.github.io/webtaskio-canirequire/#firebase) are all targeted for the client, not the server.

**Comment by kaizau** [comment]
@tehsis I know it's a big ask with holidays so close, but is there any chance of adding firebase-admin?

The clientside libraries aren't able to authenticate from Node, making it impossible to use webtask + firebase for anything that's not 100% public.

**Comment by ArtturiSipila** [comment]
Could you also add the latest Firebase version 3.6.4?

Btw, not sure what you mean by saying that clientside libraries are not able to authenticate from Node. You can use https://www.npmjs.com/package/firebase with Node or for web and the authentication should work just fine.

**Comment by kaizau** [comment]
@ArtturiSipila I get something like:

```
[10:07:20.346Z]  INFO wt:
    {
      "code": 400,
      "message": "Compilation failed: _firebase3102.default.auth(...).signInWithEmailAndPassword is not a function",
      "error": "_firebase3102.default.auth(...).signInWithEmailAndPassword is not a function",
```

Logging `firebase.auth()` confirms this:

```
[10:07:20.345Z]  INFO wt:
    { createCustomToken: [Function],
      verifyIdToken: [Function],
      INTERNAL:
       { delete: [Function],
         getToken: [Function],
         addAuthTokenListener: [Function],
         removeAuthTokenListener: [Function] } }
```

Perhaps I'm missing something? I plan on working around this with custom tokens, but it's not ideal.

**Comment by ArtturiSipila** [comment]
I tried it and yeah I'm getting similar error when running it as WebTask, locally (in Node) it seems to work though. According to this it looks valid though https://groups.google.com/forum/#!topic/firebase-talk/KzW7eAfjV5k

**Comment by alvin** [comment]
+1 for firebase-admin request

**Comment by glennblock** [comment]
We're working on it, thanks!

**Comment by glennblock** [comment]
Fixed by #56
